GPU/Vulkan/GPU_Vulkan.cpp

```diff
@@ -450,8 +445,8 @@ void GPU_Vulkan::DestroyDeviceObjects() {
 }
 
 void GPU_Vulkan::CheckRenderResized() {
+	GPUCommon::CheckRenderResized();
 	if (renderResized_) {
```

Comment:
Won't this break MSAA?  `GPUCommon::CheckRenderResized()` sets `renderResized_` to false, which means the if below will never be taken.

-[Unknown]

Reply:
Hmm, oops. I missed that, will fix